Minor fixes

hangler opened the pull request:
Hey @robinwn. Was just taking a look at this app locally to review an implementation detail and noticed a couple of issues with the most recent pull I did from master – one with the dependencies and another with the data. I've fixed them in this branch. Let me know if it works for you too.

robinwn commented:
looks like the Travis builds failed (without showing much to track down) - i don't have a container handy to dig into it.  btw, my apologies for breaking the build in master.

hangler commented:
No problem at all!

I should have mentioned this too: it looks like the build is breaking because of a kind of weird dependency bug in `create-react-app` – one of its own dependencies is requiring `babel-loader` at `8.0.5` while it itself wants `8.0.4`.\* I encountered this locally too and it is annoying. I had fixed it by editing `.env.local`, which is not in the repo, but I've also fixed it in the main `.env` file, now checked in (https://github.com/bcgov/workforce-profile-tool/pull/43/commits/01e7566d22e84b7daec323b9f0dc1065e6102bf5).

\* When I do `npm ls babel-loader`, as they suggest, I get:

```
└─┬ UNMET DEPENDENCY react-scripts@2.1.3
  ├── UNMET DEPENDENCY babel-loader@8.0.4 
  └─┬ UNMET DEPENDENCY babel-preset-react-app@7.0.1
    └── UNMET DEPENDENCY babel-loader@8.0.5 
```
